File: 프로그래머스/unrated/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    let result = 0;
    sorted = score.sort((a, b) => b - a);
    for(let i = m - 1; i < score.length; i += m) {
        result += score[i] * m;
    }
    return result;
}

File: 프로그래머스/1/135808. 과일 장수/과일 장수.py
def solution(k, m, score):
    score = sorted(score, reverse=True)
    boxes = []
    
    for i in range(0, len(score), m):
        if i + m - 1 < len(score):
            boxes.append(score[i:i + m])
    
    return sum([min(box) * m for box in boxes])